scripts/lib/mkcloud-libvirt.sh

```diff
@@ -215,10 +34,11 @@ function libvirt_start_daemon()
 
 function libvirt_setupadmin()
 {
-    libvirt_onhost_create_adminnode_config
-    libvirt_onhost_create_admin_network_config
+    onhost_local_repository_mount
```

Comment:
keep if for now, but in the long run pass the XML snippet not via a global variable but as a parameter